store/src/main/java/org/apache/rocketmq/store/ha/HAService.java

```diff
@@ -280,7 +280,9 @@ private void doWaitTransfer() {
                 if (!this.requestsRead.isEmpty()) {
                     for (CommitLog.GroupCommitRequest req : this.requestsRead) {
                         boolean transferOK = HAService.this.push2SlaveMaxOffset.get() >= req.getNextOffset();
-                        for (int i = 0; !transferOK && i < 5; i++) {
+                        long waitUntillWhen = HAService.this.defaultMessageStore.getSystemClock().now()
```

Comment:
waitUtillWhen spelling mistakes (untill-->until).

Reply:
I have corrected it.